### PR TITLE
Mongoid 5.x and Rails 4.2.x Update With RVM deploy fixes

### PR DESCRIPTION
**Pull request by hossenlopp**

### CODE REVIEW

**Review by holmesie — APPROVED** [review]

File: lib/measures/patient_builder.rb [hunk 1]
```diff
@@ -22,7 +22,7 @@ class PatientBuilder
 
     def self.rebuild_patient(patient)
 
-      patient.medical_record_number ||= Digest::MD5.hexdigest("#{patient.first} #{patient.last} #{Time.now}")
+      patient.medical_record_number ||= Digest::SHA2.hexdigest("#{patient.first} #{patient.last} #{Time.now}")
```
Comment (mulcahyk) [inline lib/measures/patient_builder.rb:25]: Just curious - why the change from MD5 to SHA2?

Comment (hossenlopp) [reply lib/measures/patient_builder.rb:25]: Brakeman complained about use of MD5 because insecurity of it. Bonnie just needs a hash of some sort for this field.

**Review by mulcahyk — APPROVED** [review]

**Comment by hossenlopp** [comment]
Rebased these changes against master and deployed to staging.

File: app/controllers/measures_controller.rb [hunk 2]
```diff
@@ -207,13 +206,20 @@ def create
         errors_dir = Rails.root.join('log', 'load_errors')
         FileUtils.mkdir_p(errors_dir)
         clean_email = File.basename(current_user.email) # Prevent path traversal
-        filename = "#{clean_email}_#{Time.now.strftime('%Y-%m-%dT%H%M%S')}#{extension}"
+        filename = "#{clean_email}_#{Time.now.strftime('%Y-%m-%dT%H%M%S')}.xmlorzip"
```
Comment (mulcahyk) [inline app/controllers/measures_controller.rb]: I'm don't understand the choice of extension here. Perhaps a quick comment would be helpful.

Comment (hossenlopp) [reply app/controllers/measures_controller.rb]: Added a comment. I needed to do this to make the security scanner happy.

**Comment by hossenlopp** [comment]
Latest changes have been deployed to staging if anyone wants to test anything on it.
